CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ feel free to ask us and community.
 * fixed increment/decrement value of embedded entity ([#3182](https://github.com/typeorm/typeorm/issues/3182))
 * fixed missing call `transformer.from()` in case column is NULL ([#3395](https://github.com/typeorm/typeorm/issues/3395))
 * fixed signatures of `update`/`insert` methods, some `find*` methods in repositories, entity managers, BaseEntity and QueryBuilders
+* fixed "deep relations" not loaded/mapped due to the built-in max length of Postgres ([#3118](https://github.com/typeorm/typeorm/issues/3118))
```

Comment:
@vlapo another issue with changelog... I think we need some other strategy...

Reply:
yeah it is pretty annoying. I think auto generate changelog from commit messages should works better. I will check more OSS projects on github and create some simple guide about it.

Reply:
Thank you